src/handlers/gestureHandlerCommon.ts

```diff
@@ -31,7 +31,10 @@ export const baseGestureHandlerProps = [
   'onHandlerStateChange',
 ] as const;
 
-export const baseGestureHandlerWithMonitorProps = commonProps;
+export const baseGestureHandlerWithMonitorProps = [
+  ...commonProps,
```

Comment:
Not clear to me why we need separate const for the and can't just add this to commonProps but I'm assuming there is good reason

Reply:
We could add it there but it's shared between gestures and gesture handlers, so the `manualActivation` prop would not be stripped from gesture handlers config if someone would add it there.